src/proc/color-formats-converter.cpp

```diff
@@ -79,147 +79,151 @@ namespace librealsense
             auto dst = reinterpret_cast<__m128i *>(d[0]);
 
 #pragma omp parallel for
-            for (int i = 0; i < n / 16; i++)
+            if (0)
```

Comment:
This function changes should be rolled back right? 
No change is expected to YUV functions